### PR TITLE
Added file dropdown feature to map creation

### PR DESCRIPTION
**Pull request by sumukhah**
fixes: #2437

Please have a detailed look before merging. Users can drag and drop on either on the map or on the Input field

Peek 2020-10-23 14-25

### CODE REVIEW

**Comment by willemarcel** [comment]
Thanks, @sumukhah! Would you mind rebasing it? Before you created that one I have made some improvements over your #3762

**Comment by sumukhah** [comment]
@willemarcel, Can you review now pls?

File: frontend/src/components/projectCreate/projectCreationMap.js [hunk 1]
```diff
@@ -50,9 +51,10 @@ const BasemapMenu = ({ map }) => {
   );
 };
 
-const ProjectCreationMap = ({ mapObj, setMapObj, metadata, updateMetadata, step }) => {
+const ProjectCreationMap = ({ mapObj, setMapObj, metadata, updateMetadata, step, uploadFile }) => {
   const mapRef = React.createRef();
   const locale = useSelector((state) => state.preferences['locale']);
+  const { getRootProps, getInputProps } = useDropzone({ onDrop: uploadFile });
```
Comment (willemarcel) [inline frontend/src/components/projectCreate/projectCreationMap.js:57]: @sumukhah you should use here the same dropzone config that I used on SetAoi. Your config doesn't allow us to draw on the map.

**Review by willemarcel — CHANGES_REQUESTED** [review]
Thanks a lot, @sumukhah. We need a little change before merging it.

**Comment by willemarcel** [comment]
@sumukhah one additional changes needed: the drag and drop upload should be enabled only on the step 1.

**Comment by willemarcel** [comment]
@sumukhah I made the change! I'll review it more carefully and merge very soon! Thanks a lot!

**Review by willemarcel — APPROVED** [review]

**Comment by sumukhah** [comment]
Thank you @willemarcel

**Pull request merged**